src/topolvm/assets/02-topolvm.yaml

```diff
@@ -83,6 +83,7 @@ kind: StorageClass
 metadata:
   name: topolvm-provisioner
   annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
```

Comment:
We also need to fix src/topolvm/generate_manifests.sh script to include this setting.